src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.js

```diff
@@ -211,7 +215,14 @@ class ContactMethodDetailsPage extends Component {
 
         const loginData = this.props.loginList[contactMethod];
         if (!contactMethod || !loginData) {
-            return <NotFoundPage />;
+            return (
+                <FullPageNotFoundView
+                    shouldShow
+                    linkKey="contacts.contactMethods"
```

Comment:
This should say `Go back to Contact Methods page`

Reply:
We should add the translation for "Go back to contact methods" to both en and es, you can ask for a Spanish translation in [#expensify-open-source](https://expensify.slack.com/archives/C01GTK53T8Q)